src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -117,17 +142,23 @@ function WorkspacePageWithSections({backButtonRoute, children, footer, guidesCal
                     guidesCallTaskID={guidesCallTaskID}
                     onBackButtonPress={() => Navigation.goBack(backButtonRoute || ROUTES.WORKSPACE_INITIAL.getRoute(policyID))}
                 />
-                {shouldUseScrollView ? (
-                    <ScrollViewWithContext
-                        keyboardShouldPersistTaps="handled"
-                        style={[styles.settingsPageBackground, styles.flex1, styles.w100]}
-                    >
-                        <View style={[styles.w100, styles.flex1]}>{content}</View>
-                    </ScrollViewWithContext>
+                {(isLoading || firstRender.current) && shouldShowLoading ? (
```

Comment:
This caused https://github.com/Expensify/App/issues/36512 where loading appears in `WorkspacePageWithSections` page when bank account process is going on